lib/fileio/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ target_sources(${PROJECT_NAME}
         writeJobContext.cpp
         chaser/chaser.cpp
         chaser/chaserRegistry.cpp
+        commands/baseExportCommand.cpp
```

Comment:
I guess this will conflict with Hamed's branch and will need to be reworked.

Reply:
Yes, I put this up as a Draft PR because it was ready for comments and figured "why wait" but always expected that I'd wait for at least Hamed's changes to go in first.